### PR TITLE
Cypress dependencies

### PR DESCRIPTION
**Pull request by Krank3n**
Added Cypress Dependencies seperately

### CODE REVIEW

File: src/js/routes.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  { path: '/notifications/alerts', name: 'Alerts', component: Alerts },
+  { path: '/notifications/badges', name: 'Badges', component: Badges },
+  { path: '/notifications/modals', name: 'Modals', component: Modals },
+  { path: '/notifications/toaster', name: 'Toaster', component: Toaster },
+  { path: '/widgets', name: 'Widgets', component: Widgets },
+  { path: '/users', exact: true,  name: 'Users', component: Users },
+  { path: '/users/:id', exact: true, name: 'User Details', component: User }
+];
+
+export default routes;
```
Comment (jakxnz) [inline src/js/routes.js:84]: Do we want these in our project? Or did that just come along for the ride unintentionally?

File: src/stories/forms.storiess.js [hunk 2]
```diff
@@ -0,0 +1,1203 @@
+import React from 'react';
```
Comment (jakxnz) [inline src/stories/forms.storiess.js:1]: Is there a spelling mistake in this file name?

**Review by jakxnz — CHANGES_REQUESTED** [review]
Cool man. I'm a little confused sorry, It seems like this PR is dedicated to getting the base cypress stuff in, but I'm not sure what it's base is. What is the base commit for these features? When I try base it on `feature/pattern-library` I get a bunch of Header component commits, and when I try base it on `feature/global-header` I don't see any code I expect. Can we do something line this up so I can see what code is related to the PR?

**Comment by Krank3n** [comment]
> Cool man. I'm a little confused sorry, it seems like this PR is dedicated to getting the base cypress stuff in, but I'm not sure what it's base is. What is the base commit for these features? When I try base it on `feature/pattern-library` I get a bunch of Header component commits, and when I try base it on `global-header` I don't see any code I expect. Can we do something line this up so I can see what code is related to the PR?

Sorry for confusion, this should be based off the pattern-library. I haved removed Header component stuff and hopefully everything else unrelated.

File: src/js/routes.js (hunk 1 above)
Comment (Krank3n) [reply src/js/routes.js]: removed

File: src/stories/forms.storiess.js (hunk 2 above)
Comment (Krank3n) [reply src/stories/forms.storiess.js]: I removed this... I think I missed spelt it on purpose so it would crash storyboard.

**Review by jakxnz — CHANGES_REQUESTED** [review]
This needs a rebase. When that's done I can give it a PR, because it should have the right scope of code then 👍

**Comment by Krank3n** [comment]
> This needs a rebase. When that's done I can give it a PR, because it should have the right scope of code then 👍

I merged dev back into this branch as a rebase was getting to complex... hopefully you are ok with that

File: cypress.env.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://underflip.frontend"
```
Comment (jakxnz) [inline cypress.env.json:2]: This isn't the base URL in my environment, could we rename this file to `cypress.env.json.dist` perhaps? Then we can each create `cypress.env.json` for our respective environments without messing with the git index

File: .gitignore [hunk 4]
```diff
@@ -3,4 +3,10 @@ public/dist
...
 .env
 coverage
-.idea/
+.idea
+
+cypress/
+
+debug.log
+
+cypress.json
```
Comment (jakxnz) [inline .gitignore:12]:
```suggestion
.idea
cypress/
debug.log
cypress.json
```

Should we be excluding `cypress.json`? Or is that a place we can store shared config, and we each use `cypress.env.json` for environment-specific config?

**Review by jakxnz — CHANGES_REQUESTED** [review]
Nice. Thanks for cleaning up the branch. Just a couple of questions about config files

File: .gitignore (hunk 4 above)
Comment (Krank3n) [reply .gitignore:12]: My cypress doesnt pick up the cypress.env.json file, needs to be cypress.json

**Review by jakxnz — APPROVED** [review]
LGTM

**Pull request merged**